tests/FakeItEasy.IntegrationTests/TypeCatalogueTests.cs

```diff
@@ -63,8 +70,7 @@ public void Should_warn_of_bad_assembly_files()
             // Arrange
             var badAssemblyFile = Path.GetTempFileName() + ".dll";
             var expectedMessageFormat =
-@"*Warning: FakeItEasy failed to load assembly '{0}' while scanning for extension points. Any IArgumentValueFormatters, IDummyFactories, and IFakeOptionsBuilders in that assembly will not be available.
-  *{0}*";
+@"*Warning: FakeItEasy failed to load assembly '{0}' while scanning for extension points. Any IArgumentValueFormatters, IDummyFactories, and IFakeOptionsBuilders in that assembly will not be available.*";

```

Comment:
The message differed slightly here, so I dropped the newline `*{0}*`. We already have the `{0}` earlier anyhow.